plugins/chat/app/services/chat/leave_channel.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
 
 module Chat
-  # Service responsible to flag a message.
+  # Service responsible for completely leaving a channel,
+  # which does something different depending on the channel:
+  #
+  # Category channels - Unfollows the channel similar to
+  # behaviour of Chat::UnfollowChannel
+  # DM channels with 2 users - Same as category channel
+  # DM channels with > 2 users (group DM) - Deletes the user's
+  # membership and removes them from the channel's user list.
```

Comment:
<img width="375" alt="Screenshot 2025-02-27 at 11 46 32 AM" src="https://github.com/user-attachments/assets/a2d5112e-2cf7-4250-935a-83a6b1437643" />

(Also noted the original comment was a copy-paste mistake. 😬)